ibm_mq/datadog_checks/ibm_mq/data/conf.yaml.example

```diff
@@ -0,0 +1,11 @@
+init_config:
+
+instances:
+  - channel: 'DEV.ADMIN.SVRCONN'
```

Comment:
Let's use the new format https://github.com/DataDog/integrations-core/blob/master/cockroachdb/datadog_checks/cockroachdb/data/conf.yaml.example

Reply:
yea I'm gonna do this

Reply:
I fixed the format a bit, but didn't change any config option